korangar/src/loaders/model/mod.rs

```diff
@@ -374,13 +430,22 @@ impl ModelLoader {
             })
             .collect();
 
-        let root_node_name = &model_data.root_node_name.clone().unwrap();
+        let mut root_node_names = Vec::<ModelString<40>>::new();
 
+        if version.equals_or_above(2, 2) {
```

Comment:
Do we also want to use a map here? In the other places with the version we also did so.